### PR TITLE
Revert "Rwilson127/test pr approval"

### PR DESCRIPTION
**Pull request by willywil548**
Reverts umgc/DevSecOpsPlayground#30

### CODE REVIEW

**Comment by jersoe** [comment]
Looks good to me

**Review by jersoe — APPROVED** [review]
Great change.

File: Blazor_Server/Shared/SubmitIdea.razor [hunk 1]
```diff
@@ -95,7 +95,6 @@
     </div>
 </div>
 
-Change
```
Comment (willywil548) [inline Blazor_Server/Shared/SubmitIdea.razor:98]: Here is an inline comment

Comment (jersoe) [reply Blazor_Server/Shared/SubmitIdea.razor:98]: Ok

**Pull request merged**